controllers/managedocs_controller.go

```diff
@@ -52,18 +53,19 @@ const (
 )
 
 const (
-	managedOCSName         = "managedocs"
-	storageClusterName     = "ocs-storagecluster"
-	prometheusName         = "managed-ocs-prometheus"
-	alertmanagerName       = "managed-ocs-alertmanager"
-	alertmanagerConfigName = "managed-ocs-alertmanager-config"
-	storageClassSizeKey    = "size"
-	deviceSetName          = "default"
-	storageClassRbdName    = "ocs-storagecluster-ceph-rbd"
-	storageClassCephFSName = "ocs-storagecluster-cephfs"
-	deployerCSVPrefix      = "ocs-osd-deployer"
-	monLabelKey            = "app"
-	monLabelValue          = "managed-ocs"
+	managedOCSName               = "managedocs"
+	storageClusterName           = "ocs-storagecluster"
+	prometheusName               = "managed-ocs-prometheus"
+	alertmanagerName             = "managed-ocs-alertmanager"
+	alertmanagerConfigSecretName = "managed-ocs-alertmanager-config-secret"
+	pagerdutySecretName          = "managed-ocs-pagerduty"
```

Comment:
This is a little bit problematic as we have multiple addons backed by the same deployer. 
The way we resolved that for the OCM parameters secret and the addon configmap is by having the secret name provided from the outside when the reconciler is created. And it turns the name of the OCM parameters secret includes the name of the addon.